.github/workflows/release_droid_release_on_maven_central.yml

```diff
@@ -23,8 +23,8 @@ jobs:
           gpg-private-key: ${{ secrets.OSSRH_GPG_SECRET_KEY }}
           gpg-passphrase: MAVEN_GPG_PASSPHRASE
       - name: Publish to Central Repository
-        run: mvn --batch-mode -Dgpg.skip=false -DskipTests clean deploy
+        run: mvn --batch-mode -Dgpg.skip=false -DskipTests clean deploy --settings maven-central-settings.xml
```

Comment:
This might not work because `actions/setup-java` writes configuration with maven central credentials to ~/.m2/settings.xml. Specifying `--settings` might override this and let maven central deployment fail.

Is it possible to specify the timeouts via command line parameters?

If this is not possible we could maybe manipulate the existing file with `sed` etc.